docs/guide/installation/06_freebsd_jail.md

```diff
@@ -115,7 +115,7 @@ Add the following to this file:
 name="zigbee2mqtt"
 rcvar=zigbee2mqtt_enable
 
-: ${fzigbee2mqtt_enable:="NO"}
+: ${zigbee2mqtt_enable:="NO"}
```

Comment:
Was this changed on purpose?

Reply:
Yes. 
Do I need to move it to another pull request?
Thank you.

Reply:
Nope, just to check, thanks!